src/AppRoot.tsx

```diff
@@ -333,6 +320,17 @@ const App: React.FC = () => {
               >
                 Export
               </Button>
+              {false && (
```

Comment:
Comment out instead?

Reply:
Choose this option because the code requires an import which would then have to be removed. Since we're planning on adding this back, I figured this was the most efficient way.